algebird-test/src/test/scala/com/twitter/algebird/LastSpec.scala

```diff
@@ -0,0 +1,24 @@
+package com.twitter.algebird
+
+import com.twitter.algebird.BaseProperties._
+import com.twitter.algebird.scalacheck.arbitrary._
+import com.twitter.algebird.scalacheck.NonEmptyVector
+import org.scalacheck.Prop.forAll
+
+class LastSpec extends CheckProperties {
+  property("Last should work properly") {
+    forAll { v: NonEmptyVector[Last[Int]] =>
+      val last = Semigroup.sumOption[Last[Int]](v.items).get
+      last == v.items.last
+    }
+  }
+
+  property("Last.aggregator returns the first item") {
```

Comment:
Ah, thank you @morazow! I'll make the change tomorrow when I add the actual documentation updates.